index.js

```diff
@@ -41,11 +41,11 @@ module.exports = {
       },
```

Comment:
Thoughts on also changing the default as well? I am still seeing lots of "Object Literal Property" fallout.

Reply:
Sure thing, I'll do that shortly.

Reply:
#86